articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,124 @@
+```
+Switch>enable
+Switch#config terminal
+Switch(config)#vlan 10
+Switch(config-vlan)#name student
+Switch(config-vlan)#exit
+Switch(config)#vlan 20
+Switch(config-vlan)#name staff
+Switch(config-vlan)#exit
+
+```
+### Step 2.
+Assign switchports to the VLANS. Ports Fa0/1 and Fa0/2 acting as Access ports for VLAN10 while ports Fa0/3 and Fa0/4 for VLAN20.The trunk port  for carrying the traffic between the two VLANS via the router we shall use Fa0/5 port.
+```
+Switch>enable
+Switch#config terminal
+Switch(config)#int fa 0/1
+Switch(config-if)#switchport mode access
+Switch(config-if)#switchport access vlan 10
+Switch(config-if)#exit
+
+
+Switch(config)#int fa 0/2
+Switch(config-if)#switchport mode access
+Switch(config-if)#switchport access vlan 10
+Switch(config-if)#exit
+
+Switch(config)#int fa 0/3
+Switch(config-if)#switchport mode access
+Switch(config-if)#switchport access vlan 20
+Switch(config-if)#exit
+
+Switch(config)#int fa 0/4
+Switch(config-if)#switchport mode access
+Switch(config-if)#switchport access vlan 20
+Switch(config-if)#exit
+
+Switch(config)#int fa 0/5
+Switch(config-if)#switchport mode trunk
+Switch(config-if)#do write
+```
+### Step 3.
+Using static  IP addressing  set the IP addresses to  each PC on the network.
+
+![IP addressing](/engineering-education/inter-vlan-routing/pc-ip-configuration.jpg)
+
+### Step 4.
+Configure the  router to enable traffic to move from VLAN10 TO VLAN20 and for the PCs to be able to communicate.This is done by subdiving the single interface on the router  into sub-interfaces which  will act as the default gateways for each of the vlans and this will allow the the two sub networks to communicate using the single interface.
+```
+Router>enable
+Router#config terminal
+Router(config)#int g0/0
+Router(config-if)#no shutdown
+Router(config-if)#int g0/0.10
+Router(config-subif)#encapsulation dot1q 10
+Router(config-subif)#ip add 192.168.1.1 255.255.255.0
+Router(config-subif)#exit
+
+Router(config)#int g0/0
+Router(config-if)#no shutdown
+Router(config-if)#int g0/0.20
+Router(config-subif)#encapsulation dot1q 20
+Router(config-subif)#ip add 192.168.2.1 255.255.255.0
+Router(config-subif)#exit
+Router(config)#do write
+Router(config)#exit
+```
+From the above configurations,interfaces g0/0 has been subdivided into two sub interfaces  g0/0.10 for VLAN 10 and g0/0.20 for VLAN 20. The two sub-interfaces are then given IP address and they serve as the trunk ports for carrying the traffic.
+### Step 5.
+Test the inter-vlan connectivity by trying to ping the different PCs .For instance if we ping PC2 in VLAN20 from PC0 in VLAN 10 it should be successful as shown below.
+
+![Ping PC2 from PC1](/engineering-education/inter-vlan-routing/ping.jpg)
+
+### Conclusion
+Inter-VLAN routing is an essential concept for any one who  has the passion for networking. It is the convenient and better way for sub dividing a large LAN and for enabling communication between the network hosts.
```

Comment:
Add a `Further reading` section by adding few links which the reader could refer for gaining more knowledge